include/ccpp_adapt.h

```diff
@@ -0,0 +1,63 @@
+
+#pragma once
+
+#include "workaround.h"
+
+#if WORKAROUND_COMPUTECPP
+
+#include <CL/sycl.hpp>
+
+/*
+This whole file is just a 'compatibility' layer for ComputeCPP until they provide full support for SYCL2020 specifications
```

Comment:
Consider adding a note as of which version this is still required (i.e., 2.5.0). Also I guess we don't need *full* 2020 support (which is probably a ways off), just for the stuff below.